tests/test_helpers/test_dependency_tree.py

```diff
@@ -36,6 +42,21 @@ def test_generation_of_dependency_tree_of_repo_packages() -> None:
     assert len(dependency_tree) > 0
 
 
+def test_generation_of_dependency_tree_of_project() -> None:
+    """Test we can generate the dependency tree of the project directory."""
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        source_path = Path(PACKAGES_DIR)
+        shutil.copytree(source_path, Path(tmp_dir) / source_path.parts[-1])
+        with cd(tmp_dir):
+            cmd = ["aea", "create", "dummy_project"]
+            project_path = Path(tmp_dir) / "dummy_project"
+            stdout, stderr = subprocess.Popen(cmd).communicate()  # nosec
+            assert stdout is stderr is None
+            dependency_tree = DependencyTree.generate(project_path, from_project=True)
+            assert len(dependency_tree) > 0
```

Comment:
Do you think we can test the tree itself too in order to make sure it is as expected?

Reply:
yes we can of course. Here there is only one item though, the signing protocol

Reply:
You can create a more complex tree using the packages in the repo.